### PR TITLE
feat(withdraw): migrated withdraw to RTK also fixed several typos in codebase

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
Migrated withdrawal to RTK and fixed several typos we had in codebase

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/data/components/actions.ts [hunk 1]
```diff
@@ -33,7 +33,8 @@ import { actions as swap } from './swap/slice'
 import * as uploadDocuments from './uploadDocuments/actions'
 import * as veriff from './veriff/actions'
 import { actions as walletConnect } from './walletConnect/slice'
-import * as withdraw from './withdraw/actions'
+// import * as withdraw from './withdraw/actions'
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/data/components/actions.ts:36]: delete

File: packages/blockchain-wallet-v4-frontend/src/data/components/withdraw/sagaRegister.ts [hunk 2]
```diff
@@ -1,24 +1,37 @@
...
     )
-    yield takeLatest(AT.FETCH_WITHDRAWAL_FEES, withdrawSagas.fetchFees)
-    yield takeLatest(AT.FETCH_WITHDRAWAL_LOCK, withdrawSagas.fetchWithdrawLocks)
-    yield takeLatest(AT.FETCH_WITHDRAWAL_CROSBSORDER_LIMITS, withdrawSagas.fetchCrossBorderLimits)
+    // yield takeLatest(AT.FETCH_WITHDRAWAL_FEES, withdrawSagas.fetchFees)
+    yield takeLatest(actions.fetchWithdrawalFees.type, withdrawSagas.fetchFees)
+    // yield takeLatest(AT.FETCH_WITHDRAWAL_LOCK, withdrawSagas.fetchWithdrawLocks)
+    yield takeLatest(actions.fetchWithdrawalLock.type, withdrawSagas.fetchWithdrawLocks)
+    // yield takeLatest(AT.FETCH_WITHDRAWAL_CROSSBORDER_LIMITS, withdrawSagas.fetchCrossBorderLimits)
+    yield takeLatest(actions.fetchCrossBorderLimits.type, withdrawSagas.fetchCrossBorderLimits)
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/data/components/withdraw/sagaRegister.ts:35]: what is the deal with all the commented out code?

File: packages/blockchain-wallet-v4-frontend/src/data/components/withdraw/slice.ts [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    handleCustodyWithdraw: (
+      state,
+      action: PayloadAction<{
+        amount: string
+        beneficiary: BeneficiaryType | BankTransferAccountType | null
+        fiatCurrency: WalletFiatType
+      }>
+    ) => {},
+    // handleWithdrawMaxAmountClick: (state, action: PayloadAction<number>) => {},
+    // handleWithdrawMinAmountClick: (state, action: PayloadAction<number>) => {},
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/data/components/withdraw/slice.ts:80]: remove?

File: packages/blockchain-wallet-v4-frontend/src/modals/BuySell/CryptoSelection/template.unsupported.tsx [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+              {'. '}
+            </>
+          ) : (
+            <>
+              <FormattedMessage
+                id='modals.simplebuy.unsupported-subcontent-1'
+                defaultMessage="Well this is awkward. We don't support buying crypto yet for"
+              />{' '}
+              <FormattedMessage id='modals.simplebuy.fiatregion' defaultMessage='your region' />
+              {'. '}
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/BuySell/CryptoSelection/template.unsupported.tsx:94]: can this just be a single formatted message?

**Review by schnogz — CHANGES_REQUESTED** [review]

File: packages/blockchain-wallet-v4-frontend/src/data/components/withdraw/sagaRegister.ts (hunk 2 above)
Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/data/components/withdraw/sagaRegister.ts:35]: ah I forgot to remove it after testing, my bad :(

File: packages/blockchain-wallet-v4-frontend/src/modals/BuySell/CryptoSelection/template.unsupported.tsx (hunk 4 above)
Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/BuySell/CryptoSelection/template.unsupported.tsx:94]: Yeah sure, this is actually some old code, will make it to be single one.

**Review by schnogz — APPROVED** [review]

**Pull request merged**